plugins/modules/events_and_notifications_workflow_manager.py

```diff
@@ -2285,8 +2284,7 @@ def get_diff_merged(self, config):
                 self.log(self.msg, "ERROR")
                 return self
 
-            pattern = re.compile(r'^[A-Za-z0-9]([A-Za-z0-9.:-]*[A-Za-z0-9])?$')
-            if server_address and not pattern.match(server_address):
+            if server_address and not self.is_valid_server_address(server_address):
                 self.status = "failed"
                 self.msg = "Invalid server adderess '{0}' given in the playbook for configuring syslog destination".format(server_address)
```

Comment:
Can you please check the spelling.
Invalid server **address**

Reply:
updated.